Can't deploy AADAdministrativeUnit - unexpected instance annotation name 'https://graph.microsoft.com/beta/administrativeUnits'

### Description of the issue

When trying to deploy an AADAdministrativeUnit we get the following: 

An unexpected instance annotation name 'https://graph.microsoft.com/beta/administrativeUnits' was found when reading from the JSON reader, In OData, Instance annotation name must start with @. 

### Microsoft 365 DSC Version

1.24.221.1

### Which workloads are affected

Azure Active Directory

### The DSC configuration

```powershell
param (
    [Parameter(Mandatory = $true)]
    [string]$ApplicationID,
    [Parameter(Mandatory = $true)]
    [string]$CertificateThumbprint,
    [Parameter(Mandatory = $true)]
    [string]$TenantName,
    [Parameter(Mandatory = $true)]
    [string]$OutputDirectory
)

Configuration AADAdministrativeUnit
{
    param (
    )

    Import-DscResource -ModuleName 'Microsoft365DSC'

    Node localhost
    {
        AADAdministrativeUnit "AADAdministrativeUnit-xyz"
        {
            ApplicationId         = $ApplicationId;
            CertificateThumbprint = $CertificateThumbprint;
            TenantId              = $TenantName;
            Description           = "description";
            DisplayName           = "xyz";
            Ensure                = "Present";
            Members               = @(
                MSFT_MicrosoftGraphMember{
                    Type = 'Group'
                    Identity = 'xyz'
                }
                MSFT_MicrosoftGraphMember{
                    Type = 'Group'
                    Identity = 'xyz'
                }
                MSFT_MicrosoftGraphMember{
                    Type = 'Group'
                    Identity = 'xyz'
                }
            );
        }
    }
}
```


### Verbose logs showing the problem

```powershell
VERBOSE: []: [[AADAdministrativeUnit]AADAdministrativeUnit-xxxx] Creating an Azure AD Administrative Unit with DisplayName {xxxx}
VERBOSE: []: [[AADAdministrativeUnit]AADAdministrativeUnit-xxxx] Creating new Administrative Unit with: description=xyz.

displayName=xyz
VERBOSE: []: [[AADAdministrativeUnit]AADAdministrativeUnit-xxxx] POST  https://graph.microsoft.com/beta/administrativeUnits with 119-byte payload
VERBOSE: []: [[AADAdministrativeUnit]AADAdministrativeUnit-xxxx] received 393-byte response of content type application/json
VERBOSE: []: [[AADAdministrativeUnit]AADAdministrativeUnit-xxxx] Adding new dynamic member {id}
##[error][Request_BadRequest] : An unexpected instance annotation name 'https://graph.microsoft.com/beta/administrativeUnits' was found when reading from the JSON reader, In OData, Instance annotation name must start with @.
##[error]PowerShell exited with code '1'.
```


### Environment Information + PowerShell Version

_No response_

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,11 +2,9 @@
 
 # UNRELEASED
 
-<<<<<<< HEAD
 * AADAdministrativeUnit
-  * Fix issue with omitted Ensure and/or Id
-    FIXES [#4437](https://github.com/microsoft/Microsoft365DSC/issues/4437)
-=======
+  * Fix issue with deploying/creating a new AU with members and/or adding members to an existing AU
+    FIXES [#4404](https://github.com/microsoft/Microsoft365DSC/issues/4404)
 * All resources
   * Fix issue where Ensure cannot be left as default 'Present'
 * AADAdministrativeUnit
@@ -28,7 +26,6 @@
 * EXOTransportRule
   * Change data type of Priority from String to Int (BREAKING)
     FIXES [[#4136](https://github.com/microsoft/Microsoft365DSC/issues/4136)]
->>>>>>> Dev
 * IntuneAppConfigurationPolicy
   * Fix comparison in Test-TargetResource
     FIXES [#4451](https://github.com/microsoft/Microsoft365DSC/issues/4451)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,9 @@
 
 # UNRELEASED
 
+* AADAdministrativeUnit
+  * Fix issue with omitted Ensure and/or Id
+    FIXES [#4437](https://github.com/microsoft/Microsoft365DSC/issues/4437)
 * IntuneAppConfigurationPolicy
   * Fix comparison in Test-TargetResource
     FIXES [#4451](https://github.com/microsoft/Microsoft365DSC/issues/4451)

--- a/Modules/Microsoft365DSC/DSCResources/MSFT_AADAdministrativeUnit/MSFT_AADAdministrativeUnit.psm1
+++ b/Modules/Microsoft365DSC/DSCResources/MSFT_AADAdministrativeUnit/MSFT_AADAdministrativeUnit.psm1
@@ -548,8 +548,8 @@ function Set-TargetResource
             foreach ($member in $memberSpecification)
             {
                 Write-Verbose -Message "Adding new dynamic member {$($member.Id)}"
+                $url = $Global:MSCloudLoginConnectionProfile.MicrosoftGraph.ResourceUrl + "beta/$($member.Type)/$($member.Id)"
                 $memberBodyParam = @{
-                    $url = $Global:MSCloudLoginConnectionProfile.MicrosoftGraph.ResourceUrl + "v1.0/$($member.Type)/$($member.Id)"
                     '@odata.id' = $url
                 }
 
@@ -646,8 +646,8 @@ function Set-TargetResource
                     {
                         Write-Verbose -Message "AdministrativeUnit {$DisplayName} Adding member {$($diff.Identity)}, type {$($diff.Type)}"
 
+                        $url = $Global:MSCloudLoginConnectionProfile.MicrosoftGraph.ResourceUrl + "beta/$memberType/$($memberObject.Id)"
                         $memberBodyParam = @{
-                            $url = $Global:MSCloudLoginConnectionProfile.MicrosoftGraph.ResourceUrl + "v1.0/$memberType/$($memberObject.Id)"
                             '@odata.id' = $url
                         }
                         New-MgBetaDirectoryAdministrativeUnitMemberByRef -AdministrativeUnitId ($currentInstance.Id) -BodyParameter $memberBodyParam | Out-Null

